docs/src/quickstart.md

```diff
@@ -137,8 +136,9 @@ julia> JuMP.primal_status(model)
 FeasiblePoint::ResultStatusCode = 1
 ```
 This indicates that GLPK has found a `FeasiblePoint` to the primal problem.
-Coupled with the `Success` from `JuMP.termination_status`, we can infer that GLPK
-has indeed found the optimal solution. We can also query `JuMP.dual_status`:
+Coupled with the `Optimal` from `JuMP.termination_status`, we can infer that
+this feasible solution is actually optimal. We can also query
```

Comment:
This is confusing. Why do we also need to check the `primal_status` if the `termination_status` was `Optimal`?